packages/themes/package.json

```diff
@@ -0,0 +1,60 @@
+{
+  "name": "@novu/themes",
+  "version": "2.0.0-canary.3",
+  "repository": "https://github.com/novuhq/novu",
+  "description": "Novu's Themes for building custom inbox notification experiences",
+  "author": "",
+  "license": "ISC",
+  "main": "dist/client/index.js",
+  "module": "dist/client/index.mjs",
+  "types": "dist/client/index.d.ts",
+  "exports": {
+    "./notion": {
+      "types": "./dist/themes/notion/index.d.ts",
+      "require": "./dist/themes/notion/index.js",
+      "import": "./dist/themes/notion/index.js"
+    }
+  },
+  "files": [
+    "dist",
+    "dist/client/**/*"
+  ],
+  "sideEffects": false,
+  "private": false,
+  "publishConfig": {
+    "access": "public"
+  },
+  "scripts": {
+    "build:watch": "tsup --watch",
+    "build": "tsup",
+    "lint": "eslint src"
+  },
+  "browserslist": {
+    "production": [
+      ">0.2%",
+      "not dead",
+      "not op_mini all"
+    ],
+    "development": [
+      "last 1 chrome version",
+      "last 1 firefox version",
+      "last 1 safari version"
+    ]
+  },
+  "devDependencies": {
+    "@types/node": "^20.14.12",
+    "@types/react": "^18.3.3",
+    "@types/react-dom": "^18.3.0",
+    "react": "^18.3.1",
+    "react-dom": "^18.3.1",
+    "tsup": "^8.2.1",
+    "typescript": "4.9.5"
+  },
+  "peerDependencies": {
+    "react": ">=17",
+    "react-dom": ">=17"
+  },
+  "dependencies": {
+    "@novu/js": "workspace:*"
```

Comment:
Should `@novu/js` be a peer dependency instead? I expect you would be using this package in conjunction with `@novu/js` and it's downstream packages like `@novu/react`

Reply:
On second thought, making `@novu/js` a peer dependency would necessitate that the consumer also installs `@novu/js`, which isn't necessary if you're using `@novu/react`.

I think instead we should add `@novu/react` and similar framework Inbox packages as `peerDependencies` and specify [`peerDependenciesMeta.<package>.optional: true`](https://docs.npmjs.com/cli/v10/configuring-npm/package-json#peerdependenciesmeta) to ensure that those peer dependencies aren't automatically installed by package managers.

Reply:
I like this approach! Will add it